data/crac/crac-api/src/main/java/com/farao_community/farao/data/crac_api/CnecLoopFlowExtension.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.data.crac_api;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+
+/**
+ * Cnec extension for loop flow
+ *
+ * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
+ */
+public class CnecLoopFlowExtension extends AbstractExtension<Cnec> {
+
+    private double loopFlowConstraint; //loop flow constraint used during optimization, set in the search tree
```

Comment:
Do not mention how/when the loop flow is set.
The data is independent from the methods. I could be set differently by another RAO implementation

Reply:
OK